aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1574,6 +1574,23 @@ Tensor reshape_symint(const Tensor& self, c10::SymIntArrayRef proposed_shape) {
   return at::_unsafe_view_symint(self.clone(at::MemoryFormat::Contiguous), shape);
 }
 
+Tensor _reshape_copy_symint(const Tensor& self, c10::SymIntArrayRef proposed_shape) {
+  if (self.is_sparse()) {
+    TORCH_CHECK(0, "_reshape_copy is not implemented for sparse tensors");
+  }
+  c10::SymDimVector shape = infer_size_dv(proposed_shape, self.sym_numel());
+
+  if (self.is_mkldnn()) {
+    TORCH_CHECK(0, "_reshape_copy not implemented for mkldnn tesnors");
+  }
+
+  if (self.is_contiguous()) {
```

Comment:
do you need this conditional, you are making a clone anyway?

Reply:
The guts here don't really matter, the inside of this function is never traced.

Reply:
Unsafe view is safe really, it does check the sizes, it just doesn't set up autograd tracking that you don't need here.

Reply:
ok, I will update this